manual/manual/cmds/top.etex

```diff
@@ -394,6 +394,25 @@ not opened, though; you still have to do
 \end{verbatim}
 yourself, if this is what you wish.
 
+\section{The native toplevel: \texttt{ocamlnat}}
+
+{\bf This section describes a tool that is not yet officially supported %
+but may be found useful.}
+
+OCaml code executing in the traditional toplevel system uses the bytecode
+interpreter.  When increased performance is required, or for testing
+programs that will only execute correctly when compiled to native code,
+the {\em native toplevel} may be used instead.
+
+From the built source tree of the OCaml distribution you may use
```

Comment:
I think that it is important to tell the reader explicitely that `ocamlnat` is not part of standard OCaml installations; and that installing it requires the compiler source files —which is of course implicit in the following, but the repetition would make more apparent the technical requirements of this section.